spot-ml/src/main/scala/org/apache/spot/dns/model/DNSSuspiciousConnectsModel.scala

```diff
@@ -186,37 +190,68 @@ object DNSSuspiciousConnectsModel {
     val dataWithWordDF = totalDataDF.withColumn(Word, dnsWordCreator.wordCreationUDF(modelColumns: _*))
 
     // aggregate per-word counts at each IP
-
-    val ipDstWordCounts =
-      dataWithWordDF.select(ClientIP, Word).map({ case Row(destIP: String, word: String) => (destIP, word) -> 1 })
-        .reduceByKey(_ + _)
-        .map({ case ((ipDst, word), count) => SpotLDACInput(ipDst, word, count) })
-
-
-    val SpotLDACOutput(ipToTopicMix, wordToPerTopicProb) = SpotLDACWrapper.runLDA(ipDstWordCounts,
-      config.modelFile,
-      config.topicDocumentFile,
-      config.topicWordFile,
-      config.mpiPreparationCmd,
-      config.mpiCmd,
-      config.mpiProcessCount,
-      config.topicCount,
-      config.localPath,
-      config.ldaPath,
-      config.localUser,
-      config.analysis,
-      config.nodes,
-      config.ldaPRGSeed)
-
-
-    new DNSSuspiciousConnectsModel(topicCount,
-      ipToTopicMix,
-      wordToPerTopicProb,
-      timeCuts,
-      frameLengthCuts,
-      subdomainLengthCuts,
-      numberPeriodsCuts,
-      entropyCuts)
+    if(config.ldaImplementation == "LDAC") {
```

Comment:
Sorry, to make sure I understand...you want me to change the names of the options and make it a switch statement (case) instead of if?